.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
If you don't think you're able to refactor the fix out then that leaves us with only one way forward from my above request.